R/lint.R

```diff
@@ -132,16 +133,19 @@ reorder_lints <- function(lints) {
 #' @param relative_path if \code{TRUE}, file paths are printed using their path
 #' relative to the package base directory.  If \code{FALSE}, use the full
 #' absolute path.
-#' @param ... additional arguments passed to \code{\link{lint}}
+#' @param ... additional arguments passed to \code{\link{lint}}, e.g. \code{cache=TRUE} or
+#' \code{exclusions=list("inst/doc/creating_linters.R" = 1, "inst/example/bad.R")}.
 #' @export
 lint_package <- function(path = ".", relative_path = TRUE, ...) {
   path <- find_package(path)
 
   read_settings(path)
   on.exit(clear_settings, add = TRUE)
 
-  names(settings$exclusions) <- normalizePath(file.path(path, names(settings$exclusions)))
-  exclusions = force(settings$exclusions)
+  args <- list(...)
```

Comment:
We should add `exclusions` to the argument list of `lint_package()` after the `...` and `#' @includeParams lint`, then modify the exclusions as needed and just use a standard call to `lint()` as before.